src/Microsoft.Azure.SignalR.Management/README.md

```diff
@@ -0,0 +1,194 @@
+# Azure SignalR Service Management SDK 
+
+## Build Status
+
+[![Travis build status](https://img.shields.io/travis/Azure/azure-signalr.svg?label=travis-ci&branch=dev&style=flat-square)](https://travis-ci.org/Azure/azure-signalr/branches)
+[![AppVeyor build status](https://img.shields.io/appveyor/ci/vicancy/azure-signalr/dev.svg?label=appveyor&style=flat-square)](https://ci.appveyor.com/project/vicancy/azure-signalr)
+
+## Nuget Packages
+
+Package Name | Target Framework | NuGet | MyGet
+[TODO]
+
+## Getting Started
+
+Azure SignalR Service Management SDK helps you to manages SignalR clients through Azure SignalR Service directly. Therefore, this SDK can be but not limited to be used in [serverless](https://azure.microsoft.com/zh-cn/solutions/serverless/) environments. You can use this SDK to manage SignalR clients connected to your Azure SignalR Service in any environment, such as in a console app, in an Azure function or in an App Server.
+
+### Features
+
+#### Transient Mode:
+
+Features are limited. The supported features are:
+
+* Publishing messages:
+  * Broadcast messages to all SignalR clients
+  * Publish messages to a specific user(s)
+  * Publish messages to a specific group(s)
+* Group membership:
+  * Add a specific user to a specific group
+  * Remove a specific user from a specific group
+
+#### Persistent Mode:
+
+Persistent mode includes the aboves fetures and are extended to all features that .Net Core SignalR supports.
```

Comment:
What does "extended to all features that .Net Core SignalR supports" mean?

Reply:
Including all features in Transient mode, but supports more, such as: 
* broadcast except some connections
* send to connections
...

 "extended to all features that .Net Core SignalR supports"  sounds user unfriendly.
I will list all supported features instead.